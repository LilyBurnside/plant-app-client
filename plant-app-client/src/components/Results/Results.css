* {
  font-family: 'Spartan';
}

.results {
  display: flex;
  align-items: center;
  flex-direction: column;
}

img {
  width: 300px;
  margin-top: 50px;
}

.plant-results {
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  background-color: #DBFFB7;
  border-radius: 15px;
  margin-top: 50px;
  width: 100%;
}

.plant-results ul {
  margin-top: 30px;
  list-style-type: none;
  padding-left: 20px;
  padding-right: 20px;
}

.plant-results li {
  padding-top: 10px;
}

.plant-results li:before {
  content: 'ðŸŒ±';
  padding-right: 5px;
}

.wishlist-button {
  background: #5A7F35;
  border: 2px solid #494949 !important;
  color: #DBFFB7 !important;
  padding: 5px 15px;
  border-radius: 15px;
  margin-left: 10px;
  text-decoration: none;
  border: none;
  margin-bottom: 20px;
}

.wishlist-button:hover {
  background-color: #D58BFA;
  transition: .25s;
}

.quiz-restart-results {
  color: #494949 !important;
  text-decoration: none;
  background: #ffffff;
  padding: 20px;
  border-radius: 15px;
  border: 4px solid #494949;
  margin-bottom: 20px;
}

.quiz-restart-results:hover {
  background-color: #D58BFA;
  color: white !important;
  transition: .25s;
}