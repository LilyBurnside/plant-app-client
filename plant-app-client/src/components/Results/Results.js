import React from 'react'
import QuizContext from '../../contexts/QuizContext'

export default class Results extends React.Component {
  
  static contextType = QuizContext

  render(){
    console.log(this.context.answers)
    return(
      <div className="results">
        <p>results page</p>
      </div>
    )
  }
}