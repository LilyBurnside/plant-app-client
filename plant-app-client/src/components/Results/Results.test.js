import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Results from './Results';

describe('Results Component', () => {

  const Context = React.createContext({ plants: [{ cmn_name: 'Pothos', sci_name: 'Epipremnum aureum', photo: 'https://plant-app-images.s3.us-east-2.amazonaws.com/epipremnum-aureum.jpg', light: 'medium', pet_safe: 'no', water: 'low', size: 'tabletop', care_level: 'beginner'}] })

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Results/>, 
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders a plant', () => {
    const wrapper = shallow(
      <Context.Provider>
        <Results />
      </Context.Provider>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  })

});