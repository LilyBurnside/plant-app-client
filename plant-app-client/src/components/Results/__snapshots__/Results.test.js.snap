// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results Component renders a plant 1`] = `<Results />`;
